<?xml version="1.0"?>
<launch>

   <!--

  "14442C109130A1D000" - 1 or blue
  "14442C10C1FAA1D000" - 2 or green
  "14442C10C1CF0FD100" - 3 or yellow
  "14442C100199A1D000" - 4 or orange

  -->

  <arg name="camera_name"          default="oak" />
  <arg name="camera_param_uri"      default="package://depthai_ros_nodes/params/camera" />
  
  <node pkg="depthai_ros_nodes" type="traffic_sign_detection_publisher.py" name="oak_traffic_sign_publisher" output="screen">
    <param name="cam_id" value="" />
    <param name="camera_name" value="$(arg camera_name)"/>
    <param name="camera_param_uri" value="$(arg camera_param_uri)"/>
    <param name="debug" value="false"/>

  </node>
  
</launch>



 
